extensions/2.0/Khronos/KHR_xmp/README.md

```diff
@@ -0,0 +1,323 @@
+| [Rational](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#rational)            | String formatted as described in the XMP documentation |
+| [FrameRate](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framerate)          | String formatted as described in the XMP documentation |
+| [FrameCount](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#framecount)        | String formatted as described in the XMP documentation |
+| [Part](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/CoreProperties.md#part)                    | String formatted as described in the XMP documentation |
+
+### Additional Properties
+
+| XMP Property Type | JSON Type |
+| ----------------- | --------- |
+| [ResourceEvent](https://github.com/adobe/xmp-docs/blob/master/XMPNamespaces/XMPDataTypes/ResourceEvent.md)| Object, we refer to the XMP documentation for a description of its properties |
+
+
+## The XMP gltf Namespace
+The `gltf` namespace is used to extend `XMP` to include metadata of specific relevance to glTF.
+The table below describes the properties in the `gltf` namespace.
+
+| Property Name  | Desrcription | JSON Type    |
+| -------------- | ------------ | ------------ |
+| spdx           | A [SPDX](https://spdx.org/licenses/) license identifier   | string |
+
+
+## Defining XMP Metadata
+An indirection level is introduced to avoid replicating the same XMP metadata in multiple glTF nodes.
+XMP metadatas are defined within a dictionary property in the glTF scene description file by adding an `extensions` property to the top-level glTF 2.0 object and defining a `KHR_xmp` property with a `packets` array inside it.
+The following example defines a glTF scene with a sample XMP metadata.
+
+```
+"extensions": {
+    "KHR_xmp" : {
+        "packets" : [
+            {
+                "dc:contributor" : [ "Creator1Name", "Creator2Email@email.com", "Creator3Name<Email@email.com>"],
+                "dc:coverage" : "Bay Area, California, United States",
+                "dc:creator" : [ "CreatorName", "CreatorEmail@email.com"],
+                "dc:date" : ["1997-07-16T19:20:30+01:00"],
+                "dc:description" : { "en-us": "text"},
+                "dc:format" : "model/gltf-binary",
+                "dc:identifier" : "urn:stock-id:292930",
+                "dc:language" : ["en"],
+                "dc:publisher" : ["Company"],
+                "dc:relation" : ["https://www.khronos.org/"],
+                "dc:rights" : {"en-us": "BSD"},
+                "dc:source" : "http://related_resource.org/derived_from_this.gltf",
+                "dc:subject" : ["architecture"],
+                "dc:title" : { "en-us" : "MyModel", "it-IT" : "Mio Modello"},
+                "dc:type" : ["Physical Object"],
+                "gltf:spdx" : "BSD-Protection",
```

Comment:
We should probably use the term "license" instead of the term "spdx", and simply state that the value must be a valid SPDX license identifier. SPDX means "Software Package Data Exchange" and includes some things beyond licenses: https://spdx.org/about.

Reply:
👍 Thanks Don, I think this is good feedback for Adam and the e-commerce WG once they will move spdx into their namespace.